src/Discord.Net.Core/Entities/Roles/Color.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 using System.Diagnostics;
-#if NETSTANDARD2_0 || NET45
+#if NETSTANDARD2_1 || NETSTANDARD2_0 || NET45
```

Comment:
Given we just merged #1382 to drop NS1.3, this conditional can be removed entirely since all supported runtimes have System.Drawing.Color now.